meta/main.yml

```diff
@@ -2,7 +2,7 @@
 galaxy_info:
     author: "Sam Doran, Josh Springer, Daniel Shepherd, James Cassell, Mike Renfro"
     description: "Apply the DISA RHEL 7 STIG"
-    company: "SSCPAC"
+    company: 
```

Comment:
This line needs to read:
`company: "MindPoint Group"`

as per this commit https://github.com/MindPointGroup/RHEL7-STIG/commit/0523e4dba87b599f9034d5455967810b315495fb It was a fix so that our import of this role on Ansible Galaxy worked correctly.